e2e/authentication/login.spec.js

```diff
@@ -0,0 +1,147 @@
+import { test, expect } from '@playwright/test';
+import { resetDatabaseAndImportDataFromPath } from '../scripts/dts-import';
+
+test.describe('Authentication | Login', () => {
+  test.beforeEach(async ({ page }) => {
+    await resetDatabaseAndImportDataFromPath({ filePath: './e2e/data/with-admin.tar' });
+    await page.goto('/admin');
+  });
+
+  test.afterEach(async ({ page }) => {
+    await page.request.fetch('/api/database/dump', {
```

Comment:
how come you still have to dump the DB if you're using DTS above?

Reply:
forgot to remove it after we managed to have a working DTS, good catch!